src/docs/PML/PMLUserGuide.adoc

```diff
@@ -1564,6 +1564,130 @@ This example trains a Mnist model, starts a service instance where the trained m
+
+There are numerous research papers and studies dedicated to analyzing the performance of categorical encoding approaches to different datasets. Based on the common factors shared by the datasets using the same encoding method, we have implemented an algorithm for finding the method best suited for your data. This can help the data scientist to start encoding smarter.
+
+To access the AutoFeat page, please follow the steps below:
+
+Open the link:https://try.activeeon.com/studio[Studio Portal].
+
+Create a new workflow.
+
+Drag and drop the <<Import_Data_Interactive>> task from the *machine-learning* bucket in the ProActive Machine Learning. The <<Import_Data_Interactive>> workflow enables users to easily import, manipulate and encode successfully their data.
+
+Click on the task and click `General Parameters` in the left to change the default parameters of this task.
+
+Put in *FILE_URL* variable the S3 link to upload your dataset.
+
+Set the other parameters according to your dataset format.
+
+Execute the workflow by setting the different workflow's variables as described in the Table below.
+
+.Import_Data_Interactive_Task variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `TASK_ENABLED`
+| If False, the task will be ignored, it will not be executed.
+| Boolean (default=True)
+| `IMPORT_FROM`
+| Selects the type of data source.
+| List [PA:URL,PA:URI,PA:USER_FILE,PA:GLOBAL_FILE] (default=PA:URL)
+| `FILE_PATH`
+| Inserts a file path/name.
+| String
+| `FILE_DELIMITER`
+| Defines a delimiter to use.
+| String (default=;)
+| `LIMIT_OUTPUT_VIEW`
+|  Specifies how many rows of the dataframe will be previewed in the browser to check each task results.
+| Int (-1 means preview all the rows)
+|===
+
+Open the link:https://try.activeeon.com/automation-dashboard/#/portal/workflow-execution[Workflow Execution Portal].
+
+You can now access the AutoFeat Page by clicking on the endpoint.
+
+You will be redirected to AutoFeat page which initially contains three tabs we describe in the following sections.
+
+=== Data Preview
+
+AutoFeat loads data from external sources. The dataset could be potentially very big. Initially, Only the 10 first data rows are displayed.
+
+The *Refresh* button enables users to see the last updates made on their data.
+[[_Data_preview]]
+image::AutoFeat_data_preview.png[align=center]
+
+=== Column summaries
+
+Whenever AutoFeat loads data from external sources, it also identifies the datatype of each column. AutoFeat does a great job at datatype recognition. Each decision can be overridden manually by the user, if required.
+
+AutoFeat also creates some summary statistics for each column. A table is displaying the missing values, minimum, maximum, mean and zeros for each numerical feature, and the cardinality (category counts) for each categorical feature.
```

Comment:
values to at the end if I well understand

Reply:
I'm not sure I understand this comment correctly. Is that abt adding values to the end of line 1647?